docs/design/repair-with-hashes.md

```diff
@@ -0,0 +1,39 @@
+# Repair using Piece Hashes
+
+## Abstract
+
+The satellite should repair files also using piece hashes to minimize CPU and bandwidth load.
+
+## Background
+
+Piece hashes Data repair is an ongoing, costly operation that will use significant bandwidth, memory, and processing power, often impacting a single operator. As a result, repair resource usage should be aggressively minimized as much as possible.
```

Comment:
doesn't look like "Piece hashes" belongs at the beginning of this first sentence